static/css/cf-enhancements.less

```diff
@@ -660,3 +660,34 @@ tbody {
     name: EOF
     eof: true
 */
+
+
+/* topdoc
+  name: Full Width Buttons
+  family: cf-enhancements
+  notes:
+    - "Modifier to buttons full width at small screen sizes"
+  patterns:
+    - name: Default example
+      markup: |
+        <btn class="btn btn__full">Button Text</btn>
+  tags:
+    - cf-buttons
```

Comment:
Or the family is `cf-enhancements`, but tagged as `cf-buttons` because the idea would be the family would be updated eventually when this migrates to CF?